fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.py

```diff
@@ -112,7 +112,7 @@ def get_context(self, context):
             "Administrator",
             self.user,
         ):
-            frappe.throw(_("Profile Not Found"), frappe.DoesNotExistError)
```

Comment:
it's not clear to me what `_` is supposed to mean here or what it is supposed to do.

Reply:
https://github.com/fossunited/fossunited/pull/602#discussion_r1757129230